source/index.md

```diff
@@ -98,4 +98,16 @@ Detailed guides on how to deploy and optimize RAPIDS.
 {bdg}`Infiniband`
 {bdg}`MIG`
 ````
+
+````{grid-item-card}
+:link: nims/index
+:link-type: doc
+{fas}`zap;sd-text-primary` NVIDIA NIMs
+^^^
+NIM Microservices using RAPIDS to accelerate your AI deployment.
+
+{bdg}`NeMo Retriever`
+{bdg}`---`
+{bdg}`---`
```

Comment:
I wonder if these labels should be categories of NIMs rather than names fospecific NIMs. For example we could add a `natural language processing` label, and a `data processing` label.

Reply:
I agree - this would help users unfamiliar with NIMs services to identify which service they might need for a particular task they have. It's totally more purpose driven this way 👍🏼